react/features/toolbox/components/Toolbox.web.js

```diff
@@ -61,6 +69,8 @@ class Toolbox extends Component {
         APP.UI.addListener(
             UIEvents.SHOW_CUSTOM_TOOLBAR_BUTTON_POPUP,
             showCustomToolbarPopup);
+
+        this.props._setDefaultToolbarButtons();
```

Comment:
I think we should add a comment why we're doing this here and a FIXME that would explain what the best solution for this action would be, i.e. adding config and interfaceConfig to the redux store and firing actions whenever we override the initial values from the URLProcessor or wherever else those values can be changed.